### DISCUSSION TITLE
how to check if alias already exists?

### DISCUSSION THREAD

**Discussion by bvandrc** [cypress-io/cypress#24289] [Questions and Help]
Is there any way to do this? `cy.get(@alias)` causes a failure when it doesn't exist, whereas I'm trying to see if it exists, without causing failure

**Comment by bvandrc** [comment]
feature request: https://github.com/cypress-io/cypress/issues/24292

**Reply by brian-mann** [reply]
Responded to you in the issue: https://github.com/cypress-io/cypress/issues/24292#issuecomment-1325648363
